specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/compute.json

```diff
@@ -0,0 +1,10988 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ComputeManagementClient",
+    "description": "The Compute Management Client.",
+    "version": "2019-12-01"
```

Comment:
this will also need to change, is that a miss?

Reply:
Yes. This is fixed now.